plugins/opa/cmd/root.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright © 2022 FairwindsOps Inc
+
+*/
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	opa "github.com/fairwindsops/insights-plugins/opa/pkg/opa"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
```

Comment:
We usually group all the third-party stuff in the middle of imports (i.e. above with logrus on line 14)